starlite/handlers/http.py

```diff
@@ -677,7 +681,10 @@ def __init__(
             security: A list of dictionaries that contain information about which security scheme can be used on the endpoint.
             summary: Text used for the route's schema summary section.
             tags: A list of string tags that will be appended to the OpenAPI schema.
+            **kwargs: Any additional kwarg - will be set in the opt dictionary.
         """
+        if "http_method" in kwargs:
+            raise ImproperlyConfiguredException("semantic route handlers cannot define http_method")
```

Comment:
SonarCloud is going to complain about using the same literal string several times instead of using a constant.

Reply:
True